<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BarangForm</class>
 <widget class="QWidget" name="BarangForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form Data Barang</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="styleSheet">
      <string notr="true">font-size:20px; font-weight:bold;</string>
     </property>
     <property name="text">
      <string>Manajemen Data Barang</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelUserAktif">
     <property name="text">
      <string>User Aktif: -</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelKode">
       <property name="text">
        <string>Kode</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="input_kode"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelNama">
       <property name="text">
        <string>Nama</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="input_nama"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelHarga">
       <property name="text">
        <string>Harga</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="input_harga"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelStok">
       <property name="text">
        <string>Stok</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="input_stok"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="btnSimpan">
       <property name="text">
        <string>Simpan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUbah">
       <property name="text">
        <string>Ubah</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnHapus">
       <property name="text">
        <string>Hapus</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBersih">
       <property name="text">
        <string>Bersih</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPrint">
       <property name="text">
        <string>Print</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="input_cari">
     <property name="placeholderText">
      <string>Cari berdasarkan kode atau nama..</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableBarang">
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Kode</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nama</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Harga</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Stok</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
